lib/model/card/card_types.dart

```diff
@@ -1,11 +1,7 @@
 import 'package:flutter/material.dart';
 
-enum CardType {
-  BASIC,
-  QUEST,
-  SPECIAL
-}
+enum CardType { BASIC, QUEST, SPECIAL }
 
-final Color basicCardColour = Colors.yellow[100];
-final Color questCardColour = Colors.purple[100];
-final Color specialCardColour = Colors.red[100];
+final Color? basicCardColour = Colors.yellow[100];
+final Color? questCardColour = Colors.purple[100];
+final Color? specialCardColour = Colors.red[100];
```

Comment:
hmm, is there a way we can make these non-nullable? They're `final`, right, and assigned a concrete value - so I think we should be able to convince the type system that they can't be null?